node1_processtime: 0.00 # millisecond
node2_processtime: 0.00 # millisecond

point_by_point_z: -10 # meter
point_by_point_r: 5.0 # meter
point_by_point_tail_theta: 45.0 # degree
point_by_point_tail_length: 10.0 # meter

voxelize: true
voxel_leaf_x: 0.2
voxel_leaf_y: 0.2
voxel_leaf_z: 0.2

denoise: true
denoise_mean_k: 50
denoise_std_dev_mult: 1.0

clustering_tolerance: 6 # meter
clustering_size_min: 30 # points number
clustering_size_max: 1000000 # points number

n_points_fov_min: 30
closest_cluster_size_min_meters: 1.5 # minimum geometric size in meters for a cluster to be considered as closest cluster

# Camera intrinsics
camera_fx0: 646.7476
camera_fy0: 645.9632
camera_cx0: 646.2584
camera_cy0: 362.9647
camera_k1_0: 0.0
camera_k2_0: 0.0
camera_p1_0: 0.0
camera_p2_0: 0.0
camera_k3_0: 0.0
camera_fx1: 644.8154
camera_fy1: 644.0333
camera_ppx1: 646.2584
camera_ppy1: 362.9647
camera_W0: 1280
camera_H0: 720
camera_W1: 1280
camera_H1: 720

# LiDAR position [x, y, z] in meters
lidar_position_x: -0.035
lidar_position_y: 0.121
lidar_position_z: 0.224

# X, Y-axis range filter values (meters)
filter_x_min: -5.0
filter_x_max: 1.0
filter_y_min: -1.0
filter_y_max: 7.0